src/screens/profile/components/career-layout/career-card.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useMemo } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { Card } from 'react-native-paper';
+import { isFunction } from 'lodash-es';
+import dayjs from 'dayjs';
+
+import {
+  CalendarIcon,
+  DeleteIcon,
+  Heading,
+  PencilIcon,
+  Text,
+} from 'src/components';
+import { AppColor, HeadingLevel, TextAppearance } from 'src/common/enums';
+import { MONTHS_IN_YEAR } from 'src/common/constants';
+import { ICareer } from './career.interface';
+import styles from './styles';
+
+type CareerProps = {
+  item: ICareer;
+  onEdit?: () => void;
+  onDelete?: () => void;
+};
+
+const CareerCard: React.FC<CareerProps> = ({ item, onEdit, onDelete }) => {
+  const { position, company, startDate, endDate } = item;
+  const endYear = dayjs(endDate).format('YYYY');
+
+  const dateString = useMemo(() => {
+    const startDay = dayjs(startDate);
+    const endDay = dayjs(endDate);
+
+    const differenceYears = endDay.diff(startDay, 'years');
+    const extraMonths = differenceYears * MONTHS_IN_YEAR;
+    const differenceMonths = endDay.diff(startDay, 'months') - extraMonths;
+
+    return `${differenceYears} yr ${differenceMonths} mo`;
+  }, [startDate, endDate]);
+
+  const handleEdit = () => {
+    if (isFunction(onEdit)) {
+      onEdit();
+    }
+    console.log('edit button clicked');
```

Comment:
```suggestion

```

Reply:
Fix in [a2ed17f](https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup-mobile/pull/27/commits/a2ed17f312e3ca0d6cbe0c295a7d978413d85eb9)